.eslintrc.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    ecmaFeatures: {
+      // Allows for the parsing of JSX
+      jsx: true,
+    },
+  },
+  ignorePatterns: [
+    'src/locales/**/*.js',
+    'src/locales/**/en-US.po',
```

Comment:
anyone have idea why this is not `*.po`? @frontend-engineers

Reply:
We can remove these `po` files, eslint won't check them